std/datetime/timezone.d

```diff
@@ -1514,9 +1530,19 @@ package:
                         west).
       +/
     static string toISOExtString(Duration utcOffset) @safe pure
+    {   
```

Comment:
Error - file 'std/datetime/timezone.d' contains trailing whitespace at line 1533
posix.mak:514: recipe for target 'checkwhitespace' failed